tests/pauli_rotations/rotation_test.py

```diff
@@ -235,6 +235,31 @@ def test_to_latex(self, input, expected):
         assert input.to_latex() == expected
 
 
+def test_cases_remove_y_operators():
+    # TODO: Add more cases here
```

Comment:
what are the 3 entries in the tuple of each case, (rotations, left_rotations, right_rotations) ?

Reply:
So we have:
* The input rotation
* The input rotation after Y operators removal is performed
* The returned `left_rotations` and `right_rotations`

Reply:
I have updated the test to follow @gwwatkin's suggestion, we should have only `input_rotation` and `output` now.